packages/language/src/preprocessor/instruction-interpreter.ts

```diff
@@ -1976,12 +1926,22 @@ async function resolveIncludeFileUri(
 
   if (pgroup) {
     // lib file as either a string or a member from a known process group
+    const absPathRegex = /^\/|\\|[A-Z]:/i;
```

Comment:
Suggestion: The `^` is only matched in the first alternative

```suggestion
    const absPathRegex = /^(\/|\\|[A-Z]:)/i;
```

Reply:
Good catch